docs/src/main/asciidoc/_attributes.adoc

```diff
@@ -67,5 +67,15 @@
 // .
 :create-app-group-id: org.acme
 :create-cli-group-id: {create-app-group-id}
+// Attributes required for single-sourcing to downstream.
+:jdk-version-other: 17
+:jdk-version-latest: 21
+:jdk-version-all: 17 or 21
```

Comment:
Let's merge this to make progress but... I don't think this will fly but it's just an intuition as I have no idea how you plan to use them.

In the community, people can use any Java 17+ (typically, if they want to use Java 23, that's fine).

So please send me an email when you start using these, with a pointer, so that I can have a look.

Reply:
Hi @gsmet,

Thanks for your review. These attributes are used mainly in the Prerequisites section of the various procedures (in the other PRs that you are reviewing/merging as part of this effort) for deploying to OpenShift. 

For example, in the Prerequisite, "You have OpenJDK {jdk-version-other} or later installed."

Per your request, I will update you via email with more details. Again, many thanks for your reviews and feedback.